chartpress.py

```diff
@@ -272,7 +282,16 @@ def _get_all_chart_paths(options):
     return list(set(paths))
 
 
-def build_image(image_spec, context_path, dockerfile_path=None, build_args=None):
+def build_image(
+    image_spec,
+    context_path,
+    dockerfile_path=None,
+    build_args=None,
+    *,
```

Comment:
It means all arguments after the `*` are keyword _only_ arguments: https://python-3-for-scientists.readthedocs.io/en/latest/python3_advanced.html (or https://www.python.org/dev/peps/pep-3102/ if you really want the hard details :smile: ).

Advantages of using it for functions with long argument lists include:
- no chance of accidentally calling the function with positional arguments in the wrong order, you _must_ pass them as keywords
- easier to refactor the function signature in future, if you want to insert a new argument or reorder the existing ones for legibility you can do so without a breaking change since the position of the arguments doesn't matter.

Reply:
Ah I like it, thanks for helping me learn about this!